pkg/ibmcloudprovider/volume_provider.go

```diff
@@ -61,15 +62,16 @@ func NewIBMCloudStorageProvider(configPath string, logger *zap.Logger) (*IBMClou
 		conf.VPC.APIVersion = "2020-07-02" // setting default values
 	}
 
+	var clusterInfo *utils.ClusterInfo
```

Comment:
can we make clusterInfo var inside the if condition

Reply:
We are using this variable outside this block also. So, had to define outside block